packit/schema.py

```diff
@@ -1,82 +1,285 @@
+import logging
+import typing
+
+from marshmallow import Schema, fields, post_load, pre_load, ValidationError
+from marshmallow_enum import EnumField
```

Comment:
I know it's WIP, but I see
```python
ModuleNotFoundError: No module named 'marshmallow_enum'
```

Reply:
strange, where did you get this error?

its configured:
https://github.com/sakalosj/packit/blob/0763eb5e6f137168aef8fd8cfb808be9c09b7bea/setup.cfg#L60-L61

also there are no errors in utni tests

Reply:
sorry, my bad